app/models/disability_claim.rb

```diff
@@ -2,5 +2,7 @@
 require_dependency 'evss/documents_service'
 
 class DisabilityClaim < ActiveRecord::Base
+  attr_accessor :successful_sync
```

Comment:
No, `attr_accessor` just defines the setter/getter methods, because there is no DB column that matches nothing will be persisted